### PR TITLE
Website readme update

### PR DESCRIPTION
**Pull request by Caleb335**
Issue:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,173 +41,181 @@ Logo made by [Harshit Sharma](https://www.github.com/icoderharshit)
...
 
 ```css
-@import '/path/to/sbuttons.less';
-
+@import "/path/to/_variables.less";
 @blue: #54a2bd; /* Make it darker */
 ```
 
-Check out [_variables.less](https://github.com/sButtons/sbuttons/blob/master/src/_variables.less) for the full list of variables.
+Check out `_variables.less` for the full list of variables.
```
Comment (shahednasser) [inline README.md:113]: Add a link to _variables.less here

**Review by shahednasser — CHANGES_REQUESTED** [review]
Great work but can you update the table of content as well?

**Comment by Caleb335** [comment]
Alright, I'd get to it.

**Comment by Caleb335** [comment]
I've made the changes you requested @shahednasser.

Kindly review

**Comment by shahednasser** [comment]
Great work as always thank you!

**Pull request merged**